src/pages/Signup/Signup.jsx

```diff
@@ -0,0 +1,18 @@
+import "./Signup.css";
+import FormSignup from "./components/FormSignup/FormSignup";

```

Comment:
A importação aqui está apontando para um arquivo que nao existe, tens que voltar 2 nivel de pasta `../../` para conseguir enxergar a pasta `components`.

1. Começamos na pasta `Signup`.
2. O primeiro `../` vai da pasta `Signup` para a pasta `pages`
3. O segundo `../` vai da pasta `pages` para a pasta `src`

```suggestion
import FormSignup from "../../components/FormSignup/FormSignup";

```